frontend/src/app/testResults/uploads/schemaBuilder.ts

```diff
@@ -0,0 +1,733 @@
+import { validate } from "uuid";
+
+export const schemaBuilder = (facilityId: string | null) => {
+  const validUuid = facilityId && validate(facilityId) ? facilityId : "";
+  const deviceCodeLookupLink = `<a href="/app/results/upload/submit/code-lookup?facility=${validUuid}" class="usa-link" target="_blank" rel="noreferrer noopener">device code lookup tool</a>`;
```

Comment:
Good catch, I tried to fix this issue too, but I was kinda a lot of work and outside the ticket scope. The app on azure is hosted on `/app`, so since this is hard-coded, it was always set to work for azure and not local env. I am happy to open another issue for it

Reply:
opened a new ticket to address the issue https://github.com/CDCgov/prime-simplereport/issues/5382

Reply:
FYI I think the fix for this would not be too bad. There is an env variable called `PUBLIC_URL` that is only setup in prod and is used to append the `/app` accordingly. [This other component](https://github.com/CDCgov/prime-simplereport/blob/5468d30911591510638104da9b2dc9d5f08b2012/frontend/src/app/signUp/IdentityVerification/Success.tsx#L13) uses it.

Reply:
oh sweet, I didn't know we were already doing something similar, will make the change

Reply:
The fix worked! thank you for the feedback